### ISSUE TITLE
[bug]: SDXL generates black images

### ISSUE THREAD

**Issue by OrbitalExplorer** [invoke-ai/InvokeAI#4454]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

10

### What version did you experience this issue on?

3.10

### What happened?

I installed sdxl with the model manager
put in a prompt
tried generating
black output (no error message)

### Screenshots

This is my result

![image](https://github.com/invoke-ai/InvokeAI/assets/75388592/1b083d07-b731-47b4-8fe5-021e63d63dd7)

### Additional context

tried with the fixed vae and without, tried 512x512 and 1024x1024, also tried multiple times reinstalling the model since it maybe was corrupted.

### Contact Details

obe.willaert@icloud.com

**Comment by Millu** [comment]
Could you try setting the VAE precision to fp32?

Or alternatively, download this VAE: https://huggingface.co/madebyollin/sdxl-vae-fp16-fix and use fp16

**Comment by Headache9940** [comment]
This is still me but GitHub said “you are unable to comment at this time” so I created a new temporary account. So here’s what I wanted to say: 
Hey there! So I tried setting the precision to fp32 and it worked (although I swear I thought I tried that before). But then I tried installing the other VAE model manually and here's what I did:

-    I dragged the VAE model into the "vae" folder in SDXL.
-    I synced the models in Invoke AI.
-    I pasted the path with the model name in the VAE section of the SDXL model card.
-    I clicked "generate" and watched the steps being taken, but after the refiner was done, I got this error.

And here's a screenshot of the error message:

![image](https://github.com/invoke-ai/InvokeAI/assets/75388592/6121fc11-5d7c-419e-a313-8f4cc8ecad62)

Thanks for the fast answer, by the way!
